test/PowerShellEditorServices.Test/Debugging/DebugServiceTests.cs

```diff
@@ -105,7 +105,7 @@ public async Task DebuggerAcceptsScriptArgs(string[] args)
             // it should not escape already escaped chars.
             ScriptFile debugWithParamsFile =
                 this.workspace.GetFile(
-                    @"..\..\..\..\PowerShellEditorServices.Test.Shared\Debugging\Debug` With Params `[Test].ps1");
+                    @"..\..\..\..\PowerShellEditorServices.Test.Shared\Debugging\Debug` W&ith Params `[Test].ps1");
```

Comment:
What was the deal with the & char?  It isn't a wildcard char.  Ah, wait a tic.  I see. On WinPS it's a reserved char and on PS Core it is used to create a (background) job.

Reply:
Yeah, it's a "token separator", so it will break a bareword string